uamqp/_connection.py

```diff
@@ -350,13 +401,20 @@ def _incoming_close(self, channel, frame):
             return
         if channel > self._channel_max:
             _LOGGER.error("Invalid channel")
```

Comment:
Looks like we might need to set `self._error` here too?

Reply:
I double checked the uamqp c logic finding out that it will close with error when meeting with this situation.
I duplicate the logic here.